core/src/com/po/fuck/model/Core.java

```diff
@@ -14,31 +14,31 @@
 import com.po.fuck.model.position.GeometryData;
 
 public class Core implements Updatable {
+
     static { // TODO: remove
         forceInit(Destructable.class);
         forceInit(All.class);
     }
-    
+
     public static Managed<Player> player;
     public static Managed<Coins> coinsCounter;
     public static Managed<ObjectFollower> objectFollower;
-    public static Assets assets;
+
     public static void initialize() {
+        Assets.load();
+        Assets.manager.finishLoading();
 
         objectFollower = Manager.create(new ObjectFollower());
-        assets = new Assets();
-        assets.load();
-        assets.manager.finishLoading();
         player = Manager.create(new Player(new GeometryData(new Vector2(),
-                                new Sprite(new Texture("FUCKerWithoutHands2.png")).getWidth(),
-                                new Sprite(new Texture("FUCKerWithoutHands2.png")).getHeight(),0)));
+                                Assets.getInfo("player").getInt("width"),
```

Comment:
Why are you using `.getInt("width")`, it can easily break, we need to add some `SpriteData` or something like this, that will guarantee that `width` and `height` and maybe some other properties are present inside this data.